packages/PriceLockup/PriceLockup.modules.scss

```diff
@@ -1,6 +1,11 @@
 @import '~@tds/core-responsive/responsive';
 @import '../../shared/scss/typography';
 
+.wrapperAlignment {
+  text-align: left;
+  display: inline-block;
```

Comment:
This fix can be achieved without `display: inline-block`

![image](https://user-images.githubusercontent.com/12798751/49092432-04f02880-f230-11e8-8c1a-45fcdebc396c.png)

Reply:
If i remove inline-block (this is when I include the code inside box component), horizontal alignment does not work from FlexGrid.Row.

Reply:
If we release this using `inline-block` then it will be a breaking change because it would cause subsequent inline elements to appear inline, for example:

![image](https://user-images.githubusercontent.com/12798751/49458467-db4c7980-f7ba-11e8-9dd4-89d2bf62c6b0.png)

```jsx
<PriceLockup
  size="medium"
  topText="Starting at"
  price="25"
  rateText="/month"
  bottomText="$68 /month after 3 months"
  signDirection="left"
/>
<ButtonLink href="#">Click me</ButtonLink>
```

We have backlogged a patch to have Price Lockup have a max width, that way the hairline divider's width won't need to be controlled by the developer via FlexGrid: https://telusdigital.atlassian.net/browse/TDS-1042